tests/publiccloud/check_registercloudguest.pm

```diff
@@ -19,7 +19,9 @@ use strict;
 use utils;
 use publiccloud::utils;
 use publiccloud::ssh_interactive 'select_host_console';
-use version_utils 'is_sle';
+
+my $path = is_sle('=15-SP2') ? '/usr/sbin/' : '';    # 15-SP2 is the oldest version that needs fullpaths
+my $regcode_param = (is_byos()) ? "-r " . get_required_var('SCC_REGCODE') : '';
```

Comment:
Using this contruction, when **NOT BYOS** all parameters are empty and the full command is unuseful, but in your below code-changes it still is executed without parameters, like: `sudo registercloudguest`.

Shouldn't we compose here, instead, the _full command_, eventually only `true` if-not-byos?

Reply:
We had a discussion with @asmorodskyi, that is the intended behavior and has worked like that so far. I am just removing one of the checks as it is impossible due to us not supporting versions that old.